runtimes/pythonrt/py-sdk/autokitteh/atlassian.py

```diff
@@ -5,7 +5,6 @@
 import os
 
 from atlassian import Confluence, Jira
-from jira import JIRA
```

Comment:
What's the different between `Jira` and `JIRA`? Who added it in the first time and why?

Reply:
I added 2 Atlassian client libs that seemed to be the most popular ones - one only for Jira (client is `jira.JIRA`) and the other is a lib for multiple Atlassian products (specifically Confluence, but also Jira - `atlassian.Jira` and `atlassian.Confluence` respectively).

After experiments and actual usage, we learned that the Jira-only lib is less suited for our auth needs, so this PR removes it in favor of using only the atlassian lib for Jira too, not just Confluence.

Reply:
OK